test/fixtures/include-path/lib-alternate/vars.scss

```diff
@@ -0,0 +1 @@
+$color: orange;
```

Comment:
`lib-alternate` might be hard to follow, what about `just keeping it as `SASS_PATH`?